services/librariesio/librariesio-sourcerank.service.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const { colorScale } = require('../../lib/color-formatters')
+const LibrariesIoBase = require('./librariesio-base')
+
+const sourceRankColor = colorScale([10, 15, 20, 25, 30])
+
+class LibrariesIoSourcerank extends LibrariesIoBase {
+  static get category() {
+    return 'other'
```

Comment:
This could also maybe sensibly live under 'dependencies'?

Reply:
I thought about that, though this is a different kind of badge. Dependencies is giving internal information about the project, rather conveying how widely used or well maintained it is.

Would it make sense to put it under Rating?

Reply:
'Rating' makes the most sense :+1: